README.md

```diff
@@ -1,2 +1,14 @@
-# TetrisGame
-A simple tetris game for learning how to use GitHub
+## Welcome to TetrisGame!
+
+- A simple tetris game for learning how to use GitHub
+- A project based learning activity for people who are getting started with Git and GitHub.
+
+To play the game navigate to: https://michelleog2351.github.io/TetrisGame/
+
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
+3. Use the left and right arrow keys to position the shape.
+4. The goal is to create complete rows with no empty spaces.
```

Comment:
I think it would be a good idea to move the goal of the game to the beginning of the instructions.